### DISCUSSION TITLE
Help to understand how to index arrays must have 1 item per line and the same for functions

### DISCUSSION THREAD

**Discussion by Maofree** [PHP-CS-Fixer/PHP-CS-Fixer#6945] [Q&A]
Hi
I'm using the "Globally (manual)" suggested here
https://cs.symfony.com/doc/installation.html

with this comand I fix all things except one

`php-cs-fixer fix ./ --rules='@PSR12,@Symfony,@PhpCsFixer'`

I want to transform this kind of array

`$friends = [1 => 'Peter', 2 => 'Paul'];`

into

![2](https://user-images.githubusercontent.com/653230/236774430-96148d7c-b0b4-45e5-af76-f6d87434ae77.png)

and the same for the function like in this example

`    public function __construct(public int $age, private string $name)
    {
     }`

![1](https://user-images.githubusercontent.com/653230/236774396-0c2b0b68-70f4-4948-ad49-850c2a284d9f.png)

in this page there aren't solutions to do that
https://mlocati.github.io/php-cs-fixer-configurator/#version:3.16

but after some research I found this page
https://github.com/symplify/coding-standard/blob/main/docs/rules_overview.md#linelengthfixer

can you help me to understand how to place parameters on multiple lines?

thanks

**Comment by Wirone** [comment]
For arrays you may be interested in #6930, but I believe we don't have fixer (or even PR ongoing) for function/method signatures 🤔.

**Comment by Maofree** [comment]
Hi
yes it is the same request of
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/4485

**Reply by Wirone** [reply]
Actually #1217 - please keep track of this issue and related PR, so you'll know when it's ready and can be used.
